app/packages/forms/controllers/form_results.coffee

```diff
@@ -0,0 +1,28 @@
+Template.form_results.onCreated ->
+  @selectedFormIdCollection = new Meteor.Collection null
+  @selectedFormIds = new ReactiveVar []
+
+Template.form_results.onRendered ->
+  instance = @
+  @autorun =>
+    _selectedFormIds =
+      _.map @selectedFormIdCollection.find().fetch(), (form) -> form.id
```

Comment:
underscore.js has a [cool function called pluck](http://underscorejs.org/#pluck) that will do this.  I think you would basically replace this line with:
`_selectedFormIds = _.pluck(@selectedFormIdCollection.find().fetch(), 'id')`